src/helpers/validators.py

```diff
@@ -0,0 +1,251 @@
+from logging import Logger
+from typing import Literal, Mapping, Optional
+
+from google.protobuf.struct_pb2 import Struct
+
+from viam.errors import ValidationError
+from viam.proto.app.robot import ComponentConfig
+
+
+from src.worker.worker import Worker
+
+
+class Validator:
```

Comment:
Any significant update to the helpers here or just moved over?

Reply:
Validators is new. The existing code that was moved was from the validate function. Now there's no real logic there for actual validations. It all takes place in `validators.py`

`helpers.py` just moves existing shared code into a single file.

Reply:
sounds good